### ISSUE TITLE
can I use this network for compression artifact reduction? and how to create training data?

### ISSUE THREAD

**Issue by maliho0803** [yulunzhang/RCAN#82]
can I use this network for compression artifact reduction? and how to create training data?

**Comment by avinabsaha** [comment]
I guess yeah. Just pass scale =1 and create datasets accordingly.

**Comment by Wangxin25** [comment]
> I guess yeah. Just pass scale =1 and create datasets accordingly.

Have you tried it? Why did I loss(L1) a lot in training?

**Comment by avinabsaha** [comment]
Yeah I have used it with scale = 1! Keep the other folder structure and other image name convention similar to scale 2/4 and it should work!

**Comment by avinabsaha** [comment]
Check out this paper for loss functions : https://arxiv.org/pdf/1511.08861.pdf

**Comment by Wangxin25** [comment]
> Check out this paper for loss functions : https://arxiv.org/pdf/1511.08861.pdf

Many thanks
